pset-functions.js

```diff
@@ -31,6 +31,31 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+  const cards = [playerCardScore, dealerCardScore]
+  const less21 = cards.filter(cards => {
+                    return cards <= 21
+                  })
+  const max = Math.max(...less21)
+
+  let result = ""
+
+  if (less21 === undefined || less21.length == 0) {
+    result = 0
+  } else {
+    result = max
+  }
+
+
+  console.log(result);
+
+}
+
+blackJack(19, 21)
```

Comment:
@jacobkelman nice job, take a look an alternative approach to this problem [found here](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L33)